Support for `access_key_metadata_writes_enabled`, `mongo_server_version` and `network_acl_bypass` in `azurerm_cosmosdb_account` (#11486)
terraform-providers/terraform-provider-azurerm: website/docs/r/cosmosdb_account.html.markdown

* `enable_multiple_write_locations` - (Optional) Enable multi-master support for this Cosmos DB account.

> * `key_based_meta_write_access_enabled` - (Optional) Is write operations on metadata resources (databases, containers, throughput) via account keys enabled? Defaults to `true`.

Reviewer: this makes little sense, by account keys does it mean the access keys? are there are ways you can write?
```suggestion
* `access_key_metadata_writes_enabled` - (Optional) Is write operations on metadata resources (databases, containers, throughput) via account keys enabled? Defaults to `true`.
```

> * `mongo_server_version` - (Optional) The Server Version of a MongoDB account. Possible values are `4.0`, `3.6`, and `3.2`. Changing this forces a new resource to be created.

Reviewer: since the default version was 3.2 before this, would it make sense to default it to 3.2 to be consistent with the previous version and remove that breaking change? and then in 3.0 we can change it to 3.6?

also would that mean the mogo index _id is no longer required if we use 3.2 here? also removing that breaking change?

Author: Since the breaking change has happened for several months, I think the users have all changed their scripts. (Since it's a breaking change from service side). Do we need to break that back?

Reviewer: didn't we only update the API now, so 2.57 is when we would start defaulting to 3.6, however if we default this to 3.2 it means there effectively would be no change?

Author: No. We only update the tests in 2.57 to adapt to the service changes. The changes have happened for several months.

> * `network_acl_bypass` - (Optional) The services which are allowed to bypass firewall checks. Possible values are `None` and `AzureServices`. Defaults to `None`.

Reviewer: will there be other vcalues here?  this may be better as
```suggestion
* `network_acl_bypass_for_azure_servers` - (Optional) If azure services can bypass ACLs. Defaults to `false`.
```

> * `network_acl_bypass_ids` - (Optional) A list of resource Ids for Network Acl Bypass for this Cosmos DB account.

Reviewer: grammer
